Soccar [Game-Server]/src/nl/soccar/gameserver/rmi/GameServerController.java

```diff
@@ -123,4 +121,28 @@ public void sessionDestroyed(String roomName) {
         }
     }
 
+    public void increaseOccupancy(String roomName) {
+        try {
+            mainServerForGameServer.increaseSessionOccupancy(gameServerForMainServer, roomName);
+        } catch (RemoteException e) {
+            LOGGER.log(Level.SEVERE, "An error occurred while reporting increased occupancy for a session.", e);
+        }
+    }
+
+    public void decreaseOccupancy(String roomName) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)